test/fixtures.py

```diff
@@ -4,27 +4,46 @@
 import logging
 import os
 import os.path
-import shutil
+import random
+import socket
+import string
 import subprocess
-import tempfile
 import time
 import uuid
 
-from six.moves import urllib
+import py
+from six.moves import urllib, xrange
```

Comment:
Personally, I prefer using the python3 syntax and then let `six` fill in the python 2 equivalent. So use `range` rather than `xrange` here--they are functionally the same.

Reply:
Looks like this got missed?

Reply:
Yep, I missed. Will take care in the upcoming PRs